chi_edge/__init__.py

```diff
@@ -15,6 +15,7 @@
 SUPPORTED_DEVICE_TYPES = (
     "raspberrypi",
     "nano",
+    "ubuntu-18.04",
```

Comment:
Does it make sense to just make this "ubuntu" and have the version optionally be set by some other host_var ?

Reply:
really, this should be something like "x86", and ansible should be getting this from the gathered facts

It really only comes up for detecting whether networking.service, networkmanager, or systemd-networkd is in use.